packages/hardhat-plugin/README.md

```diff
@@ -1,35 +1,54 @@
+![hardhat_Ignition_banner](https://github.com/NomicFoundation/hardhat-ignition/assets/24030/cc73227b-8791-4bb3-bc9a-a39be69d215f)
 [![npm](https://img.shields.io/npm/v/@nomicfoundation/hardhat-ignition.svg)](https://www.npmjs.com/package/@nomicfoundation/hardhat-ignition) [![hardhat](https://hardhat.org/buidler-plugin-badge.svg?1)](https://hardhat.org)
 
-# hardhat-ignition
+---
 
-Hardhat plugin for orchestrating deployments.
+# Hardhat Ignition
 
-## What
+Hardhat Ignition is a declarative smart contract deployment system for Ethereum. It allows you to define which smart contracts to deploy and which contract calls to make to configure them, it takes care of deployment and execution. This means focusing on what you want to do, and not in how to do it.
 
-This plugin brings **Ignition** to your **Hardhat** project, allowing you to orchestrate complex deployments to **mainnet**, to a local **Hardhat node** or within tests.
+Built by the [Nomic Foundation](https://nomic.foundation/) for the Ethereum community.
+
+Join the Hardhat Ignition channel of our [Hardhat Support Discord server](https://hardhat.org/ignition-discord) to stay up to date on new releases and tutorials.
 
 ## Installation
 
 ```bash
 npm install --save-dev @nomicfoundation/hardhat-ignition
 ```
 
-And add the following statement to your `hardhat.config.js`:
+Import the plugin in your `hardhat.config.js``:
 
 ```js
 require("@nomicfoundation/hardhat-ignition");
 ```
 
-Or, if you are using TypeScript, add this to your `hardhat.config.ts`:
+Or if you are using TypeScript, in your `hardhat.config.ts``:
 
 ```js
 import "@nomicfoundation/hardhat-ignition";
 ```
 
-## Tasks
+## Documentation
+
+On [Hardhat Ignition's website](https://hardhat.org/hardhat-ignition) you will find guides for:
+
+- [Getting started](https://hardhat.org/hardhat-ignition/getting-started)
+- [Creating Modules](https://hardhat.org/hardhat-ignition/creating-modules)
+- [Deploying a module](https://hardhat.org/hardhat-ignition/deploy)
+- [Visualizing your module](https://hardhat.org/hardhat-ignition/visualize)
+- [Handling errors](https://hardhat.org/hardhat-ignition/error-handling)
+- [Modifying an existing module](https://hardhat.org/hardhat-ignition/modifications)
+- [Using Hardhat Ignition in your tests](https://hardhat.org/hardhat-ignition/tests)
```

Comment:
These are the headings of our guides.

We could limit it to only the getting started guide. Or a subset, say: 

- getting started
- creating modules
- deploying a module

How should the links be constructed? Is it one redirect per link or is it enough that `https://hardhat.org/hardhat-ignition` redirects to x and we make the rest of the path match the structure of Ignition in the docs website (e.g. `/docs/guides/tests`?